### PR TITLE
Implement adjusted color palette to better adhere to Kiesraad brand identity

### LINKED ISSUE #528: Implement adjusted color palette to better adhere to Kiesraad brand identity
Most of the colors used semantically (errors, warnings, informational feedback) and for some other components (links, progress bars) were derived from a standard color library, to make a quick start while designing Abacus. Since then, we have created a set of semantic colors derived from the Kiesraad brand colors. These colors should be swapped in the codebase.

The colors that should be swapped can be found on [the fundamentals page in Figma](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=6081-9220&t=kfHGQ6A8QfjGbTPL-4).

Impact on our components can be seen here ([components page in Figma](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=123-2049)). "Old" colors on top, updated colors at bottom in the screenshot below.
Image

### LINKED ISSUE #602: Data-entry: row highlight missing when first input is automatically focussed
## Background
We recently implemented #503. The first input on the page is automatically focussed when navigating to a form section. When a NumberInput in our InputGrid is focussed, we also highlight the entire row (light blue background). When navigating to a form section, this highlight is not set.

## Reproduction step
(testen in chrome on macos)
- fill out a form, and click 'volgende'
- the first input of the next section is focussed. 
- expected: the row also has a highlight
- observed: the row does not have a highlight

The highlight is correctly set if you focus the next input
- use `Enter` to navigate to the next input
- expected+observed: row has highlight

When you manually move focus to the first input, it also receives input
- use `shift`+`tab` to go the the previous (first) input
- expected+observed: row has highlight

### PR DESCRIPTION
**Pull request by Lionqueen94**
Closes #528 
Closes #602 

Implemented new color palette and cleaned up variables.css, removed duplicated and unused colors

### CODE REVIEW

File: frontend/lib/ui/style/variables.css [hunk 1]
```diff
@@ -118,48 +111,48 @@
...
-  --blue-light-100: #e0f2fe;
-  --blue-light-800: #065986;
-  --menu-updates: #004eeb;
-  --link-default: #155eef;
+  --accent-orange: #ef8050;
+
+  --link-default: var(--border-color-focus);
   --link-hover: #004eeb;
   --link-active: #0040c1;
   --link-visited: #6927da;
```
Comment (Lionqueen94) [inline frontend/lib/ui/style/variables.css:121]: @jorisleker Do you have the colours for link-hover, link-active and link-visited on Figma somewhere and have these been updated also?

Comment (jorisleker) [reply frontend/lib/ui/style/variables.css:121]: @Lionqueen94 yes, they are on the [Fundamentals page](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=3087-58933&node-type=frame&t=nOtDRFnLZ9hw23eb-11). Not sure whether they were changed. I think not, but please verify!

**Review by jorisleker — CHANGES_REQUESTED** [review]
Reviewed, most looks good. 
Found 1 little issue: colors used for status on election status page don't match design. See screenshot. Design left, implementation on the right.
Scherm­afbeelding 2024-11-18 om 09 41 21

Design can be [found here](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=164-20890&t=9t8Tn2OdNZFI9XJ5-4)

**Comment by oliver3** [comment]
> Reviewed, most looks good. Found 1 little issue: colors used for status on election status page don't match design. 

Updated yellow and gray

**Review by jorisleker — APPROVED** [review]

**Review by jschuurk-kr — APPROVED** [review]

**Pull request merged**